### ISSUE TITLE
Application rename doesn't change pre-receive git hook

### ISSUE THREAD

**Issue by zenzire** [dokku/dokku#1817]
`dokku apps:rename` command doesn't change application name in hooks/pre-receive.
This causes that command `git push` to dokku doesn't trigger deploying scripts.

Reproduction procedure:

1. dokku apps:rename old_name new_name
2. change any file in the repository and commit it
3. git push dokku master

Expected results: deployment script shall be executed and new version of application shall be deployed
Results: nothing happened

**Comment by josegonzalez** [comment]
Good catch.

The patch to `plugins/apps/commands` should be the following after line 29:

```
sed -i -e "s/git-hook $OLD_APP/git-hook $NEW_APP/g" "$DOKKU_ROOT/$OLD_APP/hooks/pre-receive"
```

**Comment by josegonzalez** [comment]
This will go out in the next release :)

**Comment by natanrolnik** [comment]
Was this release already? I experienced the same today, until I enabled `dokku trace on` and saw that the pre-receive git hooks were still using the old name. I reverted back to the old name and it worked again.

**Comment by josegonzalez** [comment]
Released a *long* time ago, in 0.4.10: https://github.com/dokku/dokku/blob/master/HISTORY.md#0410

Can you file a new issue for your problem, as well as a simple repro?

**Comment by natanrolnik** [comment]
Yes, I'll try to do it soon.
